### DISCUSSION TITLE
Опрос пользователей YaMuTools

### DISCUSSION THREAD

**Discussion by Chimildic** [Chimildic/YaMuTools#52] [Беседка]
Опрос доступен по [ссылке](https://docs.google.com/forms/d/e/1FAIpQLSftXIQMD1UIXL05CN7toB3uA_pX4nAhWDyR4jNsYwBcPOOYrg/viewform?usp=sf_link)

**Comment by kisoid** [comment]
Предлагаю в 3-м вопросе добавить вариант "Не знаю".

Или добавить полутонов: "Да", "Скорее да", "Скорее нет", "Нет".

Потому что только "да" и "нет" - слишком категорично.

Например, вот прямо сейчас я пользуюсь только веб-версией ЯМузыки (и соответственно тулзов). И ближайшее время скорее всего так и будет. Поэтому ближе "нет". Но ХЗ, что там будет дальше. Вдруг я буду много ездить и мне станет удобнее мобильная версия.

Опять же, допустим я отвечу "нет" и некоторые другие тоже ответят "нет", на самом деле подразумевая "сейчас нет/не знаю". И разработчик примет на основе этого решение не делать мобильную версию. И этим самым лишит себя каких-то интересных перспектив (новый экспириенс, например). Получится как-то неправильно.

**Reply by sewyonov** [reply]
Согласен. Я выбрал вариант "Нет" по причине того, что не использую Андроид, но как будет в будущем, пока без понятия...

**Reply by kisoid** [reply]
Ну ок. Раз аффтар решил молча со мной не согласиться, проголосую выбрав из имеющейся дихотомии. Добавлю погрешности, так сказать...

**Reply by kisoid** [reply]
Над большим кол-вом вариантов ответа всегда можно было бы произвести операцию "свёртка", т.е. уменьшить кол-во вариантов путём применения весовых коэффициентов. Сделать обратное не получится...

**Comment by alex69288** [comment]
Бро, ты лучший) 
Только что думал о том чего не хватает в Яндекс Музыке, и нашел твое расширение, установил, сейчас буду тестить) но уже увидел для себя очень крутые фичи)
Спасибо)
